R/importPSIMITAB.R

```diff
@@ -0,0 +1,107 @@
+library(stringr)
+
+importNet.STRING <- function(fName,
+                             cutoffType = "xN",
+                             val,
+                             experimentType = getOptions("rete.defaultPPI"),
+                             taxID = "9606",
+                             silent = FALSE,
+                             noLog = FALSE) {
```

Comment:
It seems you are working with really, really old code. This change in the ```importNet.STRING``` code was made long, long ago and if you don't start from the most recent code base, this is going to be a **painful** journey!